core/monitor/verify.go

```diff
@@ -18,14 +18,191 @@
 package monitor
 
 import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"math/big"
+
+	"github.com/golang/glog"
+
+	"github.com/google/trillian/merkle"
+	"github.com/google/trillian/storage"
+
+	tcrypto "github.com/google/trillian/crypto"
+
+	"github.com/google/keytransparency/core/mutator/entry"
```

Comment:
Move this to the second section of the imports:

* Standard packages.
* KT packages.
* Others.
* Protos.